.gitignore

```diff
@@ -247,6 +247,7 @@ vbuild/
 /.vtools.yml
 /vtools
 /benthos
+/connect
```

Comment:
vtools logic will check out connect code at `connect/`, so need to add to `.gitignore`.

Leaving `benthos/` ignore in place... just in case - not worth any potential trouble to remove that at this point.